pages/common/pass-otp.md

```diff
@@ -0,0 +1,28 @@
+# pass-otp
+
+> A pass extension for managing one-time-password (OTP) tokens.
+> More information: <https://github.com/tadfisher/pass-otp#readme>.
+
+- Prompt for an otpauth URI token and CREATE a new pass file:
```

Comment:
The intent you use `CREATE` here is clear, but we usually do not use upper case. I prefer just use the word `create`.

What you do think? @sbrl @zdroid

Reply:
I agree, good suggestion

Reply:
Ok, I will edit APPEND too.